README.md

```diff
@@ -12,7 +12,7 @@ languages in [SonarQube](http://www.sonarqube.org/), [SonarCloud](https://sonarc
 
 ## Features
 
-* [400+ C# rules](https://rules.sonarsource.com/csharp) and [190+ VB.&#8203;NET rules](https://rules.sonarsource.com/vbnet)
+* [410+ C# rules](https://rules.sonarsource.com/csharp) and [190+ VB.&#8203;NET rules](https://rules.sonarsource.com/vbnet)

```

Comment:
When I checked the RuleTypeMappingVB.cs and RuleTypeMappingCS.cs files and searched for the regular expression `^\s*\[` I found 432 C# rules and 206 VB rules. 
Based on that: `430+ C# rules and 200+ VB.NET rules.`